includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -660,7 +610,7 @@ protected function get_wizard_shipping_methods( $country_code ) {
 		$shipping_methods = array(
 			'live_rates' => array(
 				'name'        => __( 'Live Rates', 'woocommerce' ),
-				'description' => __( 'Shipping rates updated in realtime. Powered by Jetpack.', 'woocommerce' ),
+				'description' => __( 'Live Rates powered by WooCommerce Services and Jetpack.', 'woocommerce' ),
```

Comment:
Is this enough, since the other steps used the word "install"?

Reply:
I was worried about adding too much text there because the area it's displayed is pretty tight.  Suggestions?

Reply:
Does a "Live rates are powered by WooCommerce Services and Jetpack" sentence in the shipping step header help?